src/napari_matplotlib/histogram.py

```diff
@@ -29,26 +39,143 @@ def __init__(
         parent: Optional[QWidget] = None,
     ):
         super().__init__(napari_viewer, parent=parent)
+
+        # Create widgets for setting bin parameters
+        bins_start = QDoubleSpinBox()
+        bins_start.setObjectName("bins start")
+        bins_start.setStepType(QAbstractSpinBox.AdaptiveDecimalStepType)
+        bins_start.setRange(-1e10, 1e10)
```

Comment:
I wasn't sure what a reasonable range would be (rather than the default of 0 to 100)?